### PR TITLE
Unity duplicated functions to _onMomentumScrollBeginAndEnd method

### PR DESCRIPTION
**Pull request by camelmasa**
This PR will just refactors some lines.

Please merge it if you prefer this changing 😃 
Or, Please close it if that is not good.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -152,20 +152,8 @@ const ScrollableTabView = React.createClass({
...
-          }
-        }}
-        onMomentumScrollEnd={(e) => {
-          const offsetX = e.nativeEvent.contentOffset.x;
-          const page = Math.round(offsetX / this.state.containerWidth);
-          if (this.state.currentPage !== page) {
-            this._updateSelectedPage(page);
-          }
-        }}
+        onMomentumScrollBegin={ (e) => { this._onMomentumScrollBeginAndEnd(e) }}
```
Comment (skv-headless) [inline index.js]: I think ```onMomentumScrollBegin={this._onMomentumScrollBeginAndEnd}``` would be better

Comment (camelmasa) [inline index.js]: @skv-headless Thanks for response 😃  I'll update that 👍

**Comment by skv-headless** [comment]
Sorry for delay 😞

**Comment by skv-headless** [comment]
thanks 👍

File: index.js [hunk 2]
```diff
@@ -152,20 +152,8 @@ const ScrollableTabView = React.createClass({
...
-          }
-        }}
-        onMomentumScrollEnd={(e) => {
-          const offsetX = e.nativeEvent.contentOffset.x;
-          const page = Math.round(offsetX / this.state.containerWidth);
-          if (this.state.currentPage !== page) {
-            this._updateSelectedPage(page);
-          }
-        }}
+        onMomentumScrollBegin={this._onMomentumScrollBeginAndEnd}
```
Comment (skv-headless) [inline index.js]: I've never benchmarked it but respected people says that it is good for performance 😄

**Comment by camelmasa** [comment]
@skv-headless No problem 😃  I investigated this repository's code and then I have learned about implementation and structure for react native's plugin. Thank you very much 👍

**Pull request merged**